### PR TITLE
BC-8149 - Removing Multiple Participants

### PR DESCRIPTION
**Pull request by muratmerdoglu-dp**
# Short Description

## Links to Ticket and related Pull-Requests
- https://ticketsystem.dbildungscloud.de/browse/BC-8149

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/modules/feature/room/RoomMembers/MembersTable.unit.ts [hunk 1]
```diff
@@ -66,7 +61,100 @@ describe("MembersTable", () => {
...
+			});
+		});
+
+		describe("multiple selection", () => {
+			it("should render checkBoxes", async () => {
+				const { wrapper } = setup();
+				const dataTable = wrapper.findComponent({ name: "v-data-table" });
+
+				const checkBoxes = [];
+				const rows = dataTable.findAll("tr");
```
Comment (hoeppner-dataport) [inline src/modules/feature/room/RoomMembers/MembersTable.unit.ts:74]:
```suggestion
				const checkBoxes = dataTable.findAll("tr input[type='checkbox']");
```

File: src/modules/feature/room/RoomMembers/MembersTable.unit.ts [hunk 2]
```diff
@@ -66,7 +61,100 @@ describe("MembersTable", () => {
...
+				rows.forEach((row) => {
+					const checkBox = row.find("input[type='checkbox']");
+					if (checkBox.exists()) {
+						checkBoxes.push(checkBox);
+					}
+				});
```
Comment (hoeppner-dataport) [inline src/modules/feature/room/RoomMembers/MembersTable.unit.ts:80]:
```suggestion
				const checkBoxes = dataTable.findAll("tr input[type='checkbox']");
```

File: src/modules/feature/room/RoomMembers/MembersTable.vue [hunk 3]
```diff
@@ -1,55 +1,85 @@
...
-			:mobile="null"
-			mobile-breakpoint="sm"
-			data-testid="participants-table"
-			@update:current-items="onUpdateFilter"
-		>
-			<template #[`item.actions`]="{ item }">
+	<div
+		class="d-flex justify-space-between align-center mb-2 table-title-header"
+	>
+		<div class="mr-2 pa-0 pl-4 multi-action-menu">
```
Comment (hoeppner-dataport) [inline src/modules/feature/room/RoomMembers/MembersTable.vue:5]: I was wondering if this couldn't be some kind of existing vuetifyJs-Component... Especially as it needs some CSS. But I assume you checked and did not find anything suiting.

**Review by hoeppner-dataport — DISMISSED** [review]

File: src/modules/feature/room/RoomMembers/MembersTable.vue (hunk 3 above)
Comment (muratmerdoglu-dp) [reply src/modules/feature/room/RoomMembers/MembersTable.vue:5]: Yes, I tried to use the Vuetify Banner component, but it needed some custom overwriting on CSS to adjust the items' alignments, so I gave up using it.

**Review by hoeppner-dataport — APPROVED** [review]

**Review by hoeppner-dataport — APPROVED** [review]

**Pull request merged**